cg/meta/transfer/external_data.py

```diff
@@ -60,17 +60,28 @@ def get_destination_path(self, customer: str, lims_sample_id: Optional[str] = ""
         """Returns the path to where the files are to be transferred"""
         return Path(self.destination_path % customer, lims_sample_id)
 
-    def transfer_sample_files_from_source(self, dry_run: bool, ticket_id: int) -> None:
+    def transfer_sample_files_from_source(
+        self, dry_run: bool, nanopore: bool, ticket_id: int
+    ) -> None:
         """Transfers all sample files, related to given ticket, from source to destination"""
         cust: str = self.status_db.get_customer_id_from_ticket(ticket_id=ticket_id)
         log_dir: Path = self.create_log_dir(ticket_id=ticket_id, dry_run=dry_run)
         error_function: str = ERROR_RSYNC_FUNCTION.format()
-        Path(self.destination_path % cust).mkdir(exist_ok=True)
 
-        command: str = RSYNC_CONTENTS_COMMAND.format(
-            source_path=self.get_source_path(customer=cust, ticket_id=ticket_id),
-            destination_path=self.get_destination_path(customer=cust),
-        )
+        destination_folder: Path = Path(self.destination_path % cust)
+        if nanopore:
+            destination_folder = destination_folder / "nanopore"
+            command: str = RSYNC_COMMAND.format(
+                source_path=self.get_source_path(customer=cust, ticket_id=ticket_id),
+                destination_path=destination_folder,
+            )
+        else:
+            command: str = RSYNC_CONTENTS_COMMAND.format(
```

Comment:
And then another function rsync_command() to be consistent

Reply:
look at how I solved it and let me know what you think, I am not sure I understand how you want it.

Reply:
I think i meant keeping the if statement in the     def transfer_sample_files_from_source() and have one function for each command type. But this looks fine to me!